content/docs/user-guide/scripting/lua/debugging-tutorial.md

```diff
@@ -100,7 +102,9 @@ You may need to expand the Lua Editor window to see the buttons on the Lua Edito
 The class reference feature is active only for the default context and component entity scripts.
 {{< /note >}}
 
-    After you connect, you can pause the execution of a given script by setting breakpoints.
+## Setting Breakpoints
```

Comment:
I think breakpoints is used as a generic term in this context, therefore no capitalization.
```suggestion
## Setting breakpoints
```